{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import qiime2 as q2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1A_GUT_26_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1A_GUT_26_93_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1A_GUT_26_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1A_GUT_26_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1A_GUT_26_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1A_GUT_26_93_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1A_GUT_26_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1A_GUT_26_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       118,606 reads  @     29.0 µs/read;   2.07 M reads/minute\n",
      "Finished in 3.48 s (29 us/read; 2.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            118,606\n",
      "  Read 1 with adapter:                       2 (0.0%)\n",
      "  Read 2 with adapter:                     279 (0.2%)\n",
      "Pairs that were too short:                  81 (0.1%)\n",
      "Pairs written (passing filters):       118,525 (99.9%)\n",
      "\n",
      "Total basepairs processed:    59,540,212 bp\n",
      "  Read 1:    29,770,106 bp\n",
      "  Read 2:    29,770,106 bp\n",
      "Total written (filtered):     59,498,684 bp (99.9%)\n",
      "  Read 1:    29,749,769 bp\n",
      "  Read 2:    29,748,915 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 2 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1853.2\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 279 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t189\t1853.2\t0\t189\n",
      "5\t1\t115.8\t0\t1\n",
      "36\t8\t0.0\t3\t0 0 3 5\n",
      "37\t54\t0.0\t3\t0 3 10 41\n",
      "38\t27\t0.0\t3\t0 2 8 17\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_CF_26_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_CF_26_75_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_CF_26_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_CF_26_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_CF_26_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_CF_26_75_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_CF_26_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_CF_26_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:08       261,810 reads  @     32.6 µs/read;   1.84 M reads/minute\n",
      "Finished in 8.59 s (33 us/read; 1.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            261,810\n",
      "  Read 1 with adapter:                     256 (0.1%)\n",
      "  Read 2 with adapter:                      74 (0.0%)\n",
      "Pairs that were too short:                  28 (0.0%)\n",
      "Pairs written (passing filters):       261,782 (100.0%)\n",
      "\n",
      "Total basepairs processed:   131,428,620 bp\n",
      "  Read 1:    65,714,310 bp\n",
      "  Read 2:    65,714,310 bp\n",
      "Total written (filtered):    131,413,554 bp (100.0%)\n",
      "  Read 1:    65,706,513 bp\n",
      "  Read 2:    65,707,041 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 256 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t255\t4090.8\t0\t255\n",
      "4\t1\t1022.7\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 74 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t38\t4090.8\t0\t38\n",
      "4\t5\t1022.7\t0\t5\n",
      "35\t1\t0.0\t3\t0 0 0 1\n",
      "36\t2\t0.0\t3\t0 0 0 2\n",
      "37\t13\t0.0\t3\t0 0 2 11\n",
      "38\t15\t0.0\t3\t0 0 1 14\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_CF_30_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_CF_30_70_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_CF_30_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_CF_30_70_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_CF_30_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_CF_30_70_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_CF_30_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_CF_30_70_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:07       258,673 reads  @     30.8 µs/read;   1.95 M reads/minute\n",
      "Finished in 8.02 s (31 us/read; 1.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            258,673\n",
      "  Read 1 with adapter:                     234 (0.1%)\n",
      "  Read 2 with adapter:                      53 (0.0%)\n",
      "Pairs that were too short:                  21 (0.0%)\n",
      "Pairs written (passing filters):       258,652 (100.0%)\n",
      "\n",
      "Total basepairs processed:   129,853,846 bp\n",
      "  Read 1:    64,926,923 bp\n",
      "  Read 2:    64,926,923 bp\n",
      "Total written (filtered):    129,842,392 bp (100.0%)\n",
      "  Read 1:    64,920,950 bp\n",
      "  Read 2:    64,921,442 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 234 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t234\t4041.8\t0\t234\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 53 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t4041.8\t0\t26\n",
      "5\t1\t252.6\t0\t1\n",
      "9\t1\t1.0\t0\t1\n",
      "10\t1\t0.2\t1\t0 1\n",
      "36\t3\t0.0\t3\t0 0 1 2\n",
      "37\t12\t0.0\t3\t0 1 3 8\n",
      "38\t9\t0.0\t3\t0 0 1 8\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_DP_26_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_DP_26_81_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_DP_26_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_DP_26_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_DP_26_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_DP_26_81_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_DP_26_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_DP_26_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       150,381 reads  @     32.5 µs/read;   1.85 M reads/minute\n",
      "Finished in 4.92 s (33 us/read; 1.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            150,381\n",
      "  Read 1 with adapter:                     153 (0.1%)\n",
      "  Read 2 with adapter:                      80 (0.1%)\n",
      "Pairs that were too short:                  65 (0.0%)\n",
      "Pairs written (passing filters):       150,316 (100.0%)\n",
      "\n",
      "Total basepairs processed:    75,491,262 bp\n",
      "  Read 1:    37,745,631 bp\n",
      "  Read 2:    37,745,631 bp\n",
      "Total written (filtered):     75,457,992 bp (100.0%)\n",
      "  Read 1:    37,728,860 bp\n",
      "  Read 2:    37,729,132 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 153 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t152\t2349.7\t0\t152\n",
      "222\t1\t0.0\t6\t0 0 0 0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 80 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t2349.7\t0\t10\n",
      "5\t1\t146.9\t0\t1\n",
      "6\t1\t36.7\t0\t1\n",
      "35\t1\t0.0\t3\t0 0 0 1\n",
      "36\t3\t0.0\t3\t0 0 1 2\n",
      "37\t29\t0.0\t3\t0 3 5 21\n",
      "38\t34\t0.0\t3\t1 1 9 23\n",
      "39\t1\t0.0\t3\t0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_DP_30_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_DP_30_78_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_DP_30_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_DP_30_78_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_DP_30_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_DP_30_78_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_DP_30_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_DP_30_78_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       166,724 reads  @     28.0 µs/read;   2.14 M reads/minute\n",
      "Finished in 4.71 s (28 us/read; 2.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            166,724\n",
      "  Read 1 with adapter:                     165 (0.1%)\n",
      "  Read 2 with adapter:                      53 (0.0%)\n",
      "Pairs that were too short:                  39 (0.0%)\n",
      "Pairs written (passing filters):       166,685 (100.0%)\n",
      "\n",
      "Total basepairs processed:    83,695,448 bp\n",
      "  Read 1:    41,847,724 bp\n",
      "  Read 2:    41,847,724 bp\n",
      "Total written (filtered):     83,675,166 bp (100.0%)\n",
      "  Read 1:    41,837,440 bp\n",
      "  Read 2:    41,837,726 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 165 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t165\t2605.1\t0\t165\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 53 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t2605.1\t0\t8\n",
      "6\t1\t40.7\t0\t1\n",
      "35\t1\t0.0\t3\t0 0 0 1\n",
      "36\t4\t0.0\t3\t0 0 1 3\n",
      "37\t25\t0.0\t3\t0 1 8 16\n",
      "38\t14\t0.0\t3\t0 0 4 10\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_Feces_26_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_Feces_26_65_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_Feces_26_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_Feces_26_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_Feces_26_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_Feces_26_65_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_Feces_26_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_Feces_26_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       200,634 reads  @     30.9 µs/read;   1.94 M reads/minute\n",
      "Finished in 6.24 s (31 us/read; 1.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            200,634\n",
      "  Read 1 with adapter:                     228 (0.1%)\n",
      "  Read 2 with adapter:                     188 (0.1%)\n",
      "Pairs that were too short:                  16 (0.0%)\n",
      "Pairs written (passing filters):       200,618 (100.0%)\n",
      "\n",
      "Total basepairs processed:   100,718,268 bp\n",
      "  Read 1:    50,359,134 bp\n",
      "  Read 2:    50,359,134 bp\n",
      "Total written (filtered):    100,708,858 bp (100.0%)\n",
      "  Read 1:    50,354,434 bp\n",
      "  Read 2:    50,354,424 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 228 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t228\t3134.9\t0\t228\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 188 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t127\t3134.9\t0\t127\n",
      "4\t38\t783.7\t0\t38\n",
      "6\t3\t49.0\t0\t3\n",
      "35\t1\t0.0\t3\t0 0 0 1\n",
      "36\t3\t0.0\t3\t0 0 0 3\n",
      "37\t9\t0.0\t3\t0 0 2 7\n",
      "38\t7\t0.0\t3\t0 0 1 6\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_Feces_30_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_Feces_30_59_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_Feces_30_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_Feces_30_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_Feces_30_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_Feces_30_59_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_Feces_30_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_Feces_30_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       170,892 reads  @     31.6 µs/read;   1.90 M reads/minute\n",
      "Finished in 5.45 s (32 us/read; 1.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            170,892\n",
      "  Read 1 with adapter:                     147 (0.1%)\n",
      "  Read 2 with adapter:                      44 (0.0%)\n",
      "Pairs that were too short:                   5 (0.0%)\n",
      "Pairs written (passing filters):       170,887 (100.0%)\n",
      "\n",
      "Total basepairs processed:    85,787,784 bp\n",
      "  Read 1:    42,893,892 bp\n",
      "  Read 2:    42,893,892 bp\n",
      "Total written (filtered):     85,784,602 bp (100.0%)\n",
      "  Read 1:    42,892,196 bp\n",
      "  Read 2:    42,892,406 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 147 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t147\t2670.2\t0\t147\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 44 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t2670.2\t0\t27\n",
      "4\t5\t667.5\t0\t5\n",
      "5\t1\t166.9\t0\t1\n",
      "6\t3\t41.7\t0\t3\n",
      "35\t1\t0.0\t3\t0 0 0 1\n",
      "36\t2\t0.0\t3\t0 0 0 2\n",
      "37\t4\t0.0\t3\t0 1 1 2\n",
      "38\t1\t0.0\t3\t0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_Feed_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_Feed_84_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_Feed_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_Feed_84_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_Feed_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_Feed_84_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_Feed_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_Feed_84_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:06       208,052 reads  @     32.9 µs/read;   1.82 M reads/minute\n",
      "Finished in 6.90 s (33 us/read; 1.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            208,052\n",
      "  Read 1 with adapter:                     157 (0.1%)\n",
      "  Read 2 with adapter:                      40 (0.0%)\n",
      "Pairs that were too short:                   8 (0.0%)\n",
      "Pairs written (passing filters):       208,044 (100.0%)\n",
      "\n",
      "Total basepairs processed:   104,442,104 bp\n",
      "  Read 1:    52,221,052 bp\n",
      "  Read 2:    52,221,052 bp\n",
      "Total written (filtered):    104,437,485 bp (100.0%)\n",
      "  Read 1:    52,218,573 bp\n",
      "  Read 2:    52,218,912 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 157 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t157\t3250.8\t0\t157\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 40 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t3250.8\t0\t30\n",
      "6\t1\t50.8\t0\t1\n",
      "36\t1\t0.0\t3\t0 0 0 1\n",
      "37\t1\t0.0\t3\t0 0 0 1\n",
      "38\t7\t0.0\t3\t0 0 1 6\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_GUT_30_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_GUT_30_89_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_GUT_30_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_GUT_30_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_GUT_30_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_GUT_30_89_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_GUT_30_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_GUT_30_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       198,874 reads  @     25.3 µs/read;   2.37 M reads/minute\n",
      "Finished in 5.08 s (26 us/read; 2.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            198,874\n",
      "  Read 1 with adapter:                       8 (0.0%)\n",
      "  Read 2 with adapter:                     557 (0.3%)\n",
      "Pairs that were too short:                  44 (0.0%)\n",
      "Pairs written (passing filters):       198,830 (100.0%)\n",
      "\n",
      "Total basepairs processed:    99,834,748 bp\n",
      "  Read 1:    49,917,374 bp\n",
      "  Read 2:    49,917,374 bp\n",
      "Total written (filtered):     99,810,605 bp (100.0%)\n",
      "  Read 1:    49,906,306 bp\n",
      "  Read 2:    49,904,299 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 8 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t3107.4\t0\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 557 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t497\t3107.4\t0\t497\n",
      "5\t1\t194.2\t0\t1\n",
      "35\t5\t0.0\t3\t0 0 0 5\n",
      "36\t10\t0.0\t3\t0 0 2 8\n",
      "37\t27\t0.0\t3\t0 1 4 22\n",
      "38\t16\t0.0\t3\t0 0 5 11\n",
      "39\t1\t0.0\t3\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_SWTR_26_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_SWTR_26_54_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_SWTR_26_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_SWTR_26_54_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_SWTR_26_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_SWTR_26_54_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_SWTR_26_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_SWTR_26_54_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:08       257,650 reads  @     31.8 µs/read;   1.89 M reads/minute\n",
      "Finished in 8.25 s (32 us/read; 1.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            257,650\n",
      "  Read 1 with adapter:                     259 (0.1%)\n",
      "  Read 2 with adapter:                     152 (0.1%)\n",
      "Pairs that were too short:                 106 (0.0%)\n",
      "Pairs written (passing filters):       257,544 (100.0%)\n",
      "\n",
      "Total basepairs processed:   129,340,300 bp\n",
      "  Read 1:    64,670,150 bp\n",
      "  Read 2:    64,670,150 bp\n",
      "Total written (filtered):    129,285,752 bp (100.0%)\n",
      "  Read 1:    64,642,770 bp\n",
      "  Read 2:    64,642,982 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 259 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t259\t4025.8\t0\t259\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 152 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t4025.8\t0\t33\n",
      "35\t5\t0.0\t3\t0 0 0 5\n",
      "36\t8\t0.0\t3\t0 0 1 7\n",
      "37\t52\t0.0\t3\t0 5 14 33\n",
      "38\t54\t0.0\t3\t0 2 16 36\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_SWTR_30_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_SWTR_30_50_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_SWTR_30_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_SWTR_30_50_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_SWTR_30_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR1_SWTR_30_50_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_SWTR_30_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR1_SWTR_30_50_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:06       221,105 reads  @     31.3 µs/read;   1.92 M reads/minute\n",
      "Finished in 6.96 s (31 us/read; 1.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            221,105\n",
      "  Read 1 with adapter:                     207 (0.1%)\n",
      "  Read 2 with adapter:                     123 (0.1%)\n",
      "Pairs that were too short:                 104 (0.0%)\n",
      "Pairs written (passing filters):       221,001 (100.0%)\n",
      "\n",
      "Total basepairs processed:   110,994,710 bp\n",
      "  Read 1:    55,497,355 bp\n",
      "  Read 2:    55,497,355 bp\n",
      "Total written (filtered):    110,941,396 bp (100.0%)\n",
      "  Read 1:    55,470,630 bp\n",
      "  Read 2:    55,470,766 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 207 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t207\t3454.8\t0\t207\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 123 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t3454.8\t0\t6\n",
      "35\t1\t0.0\t3\t0 0 0 1\n",
      "36\t12\t0.0\t3\t0 0 3 9\n",
      "37\t44\t0.0\t3\t0 5 10 29\n",
      "38\t60\t0.0\t3\t1 4 10 45\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2A_GUT_26_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2A_GUT_26_92_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2A_GUT_26_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2A_GUT_26_92_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2A_GUT_26_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2A_GUT_26_92_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2A_GUT_26_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2A_GUT_26_92_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       146,777 reads  @     27.1 µs/read;   2.21 M reads/minute\n",
      "Finished in 4.02 s (27 us/read; 2.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            146,777\n",
      "  Read 1 with adapter:                       9 (0.0%)\n",
      "  Read 2 with adapter:                     261 (0.2%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       146,776 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,682,054 bp\n",
      "  Read 1:    36,841,027 bp\n",
      "  Read 2:    36,841,027 bp\n",
      "Total written (filtered):     73,680,611 bp (100.0%)\n",
      "  Read 1:    36,840,747 bp\n",
      "  Read 2:    36,839,864 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 9 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t2293.4\t0\t7\n",
      "4\t2\t573.3\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 261 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t255\t2293.4\t0\t255\n",
      "4\t1\t573.3\t0\t1\n",
      "35\t1\t0.0\t3\t0 0 0 1\n",
      "36\t3\t0.0\t3\t0 0 0 3\n",
      "37\t1\t0.0\t3\t0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_CF_26_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_CF_26_74_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_CF_26_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_CF_26_74_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_CF_26_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_CF_26_74_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_CF_26_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_CF_26_74_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       170,639 reads  @     26.0 µs/read;   2.31 M reads/minute\n",
      "Finished in 4.48 s (26 us/read; 2.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            170,639\n",
      "  Read 1 with adapter:                      17 (0.0%)\n",
      "  Read 2 with adapter:                      33 (0.0%)\n",
      "Pairs that were too short:                  15 (0.0%)\n",
      "Pairs written (passing filters):       170,624 (100.0%)\n",
      "\n",
      "Total basepairs processed:    85,660,778 bp\n",
      "  Read 1:    42,830,389 bp\n",
      "  Read 2:    42,830,389 bp\n",
      "Total written (filtered):     85,653,110 bp (100.0%)\n",
      "  Read 1:    42,826,573 bp\n",
      "  Read 2:    42,826,537 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 17 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t2666.2\t0\t17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 33 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t2666.2\t0\t17\n",
      "36\t1\t0.0\t3\t0 0 0 1\n",
      "37\t5\t0.0\t3\t0 0 2 3\n",
      "38\t10\t0.0\t3\t0 0 2 8\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_CF_30_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_CF_30_69_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_CF_30_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_CF_30_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_CF_30_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_CF_30_69_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_CF_30_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_CF_30_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:07       243,202 reads  @     30.8 µs/read;   1.95 M reads/minute\n",
      "Finished in 7.54 s (31 us/read; 1.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            243,202\n",
      "  Read 1 with adapter:                     246 (0.1%)\n",
      "  Read 2 with adapter:                      78 (0.0%)\n",
      "Pairs that were too short:                  49 (0.0%)\n",
      "Pairs written (passing filters):       243,153 (100.0%)\n",
      "\n",
      "Total basepairs processed:   122,087,404 bp\n",
      "  Read 1:    61,043,702 bp\n",
      "  Read 2:    61,043,702 bp\n",
      "Total written (filtered):    122,061,868 bp (100.0%)\n",
      "  Read 1:    61,030,665 bp\n",
      "  Read 2:    61,031,203 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 246 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t246\t3800.0\t0\t246\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 78 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t3800.0\t0\t9\n",
      "4\t7\t950.0\t0\t7\n",
      "5\t10\t237.5\t0\t10\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "36\t2\t0.0\t3\t0 0 1 1\n",
      "37\t16\t0.0\t3\t0 1 2 13\n",
      "38\t31\t0.0\t3\t0 1 12 18\n",
      "39\t1\t0.0\t3\t0 0 0 1\n",
      "41\t1\t0.0\t4\t0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_DP_26_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_DP_26_80_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_DP_26_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_DP_26_80_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_DP_26_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_DP_26_80_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_DP_26_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_DP_26_80_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       143,405 reads  @     31.6 µs/read;   1.90 M reads/minute\n",
      "Finished in 4.58 s (32 us/read; 1.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            143,405\n",
      "  Read 1 with adapter:                      50 (0.0%)\n",
      "  Read 2 with adapter:                      35 (0.0%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       143,404 (100.0%)\n",
      "\n",
      "Total basepairs processed:    71,989,310 bp\n",
      "  Read 1:    35,994,655 bp\n",
      "  Read 2:    35,994,655 bp\n",
      "Total written (filtered):     71,988,549 bp (100.0%)\n",
      "  Read 1:    35,994,254 bp\n",
      "  Read 2:    35,994,295 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 50 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t2240.7\t0\t50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 35 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t2240.7\t0\t31\n",
      "4\t1\t560.2\t0\t1\n",
      "6\t2\t35.0\t0\t2\n",
      "37\t1\t0.0\t3\t0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_DP_30_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_DP_30_77_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_DP_30_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_DP_30_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_DP_30_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_DP_30_77_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_DP_30_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_DP_30_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       175,036 reads  @     30.0 µs/read;   2.00 M reads/minute\n",
      "Finished in 5.28 s (30 us/read; 1.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            175,036\n",
      "  Read 1 with adapter:                     174 (0.1%)\n",
      "  Read 2 with adapter:                      62 (0.0%)\n",
      "Pairs that were too short:                  51 (0.0%)\n",
      "Pairs written (passing filters):       174,985 (100.0%)\n",
      "\n",
      "Total basepairs processed:    87,868,072 bp\n",
      "  Read 1:    43,934,036 bp\n",
      "  Read 2:    43,934,036 bp\n",
      "Total written (filtered):     87,841,785 bp (100.0%)\n",
      "  Read 1:    43,920,713 bp\n",
      "  Read 2:    43,921,072 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 174 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t174\t2734.9\t0\t174\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 62 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t2734.9\t0\t6\n",
      "16\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t3\t0 0 0 1\n",
      "36\t2\t0.0\t3\t0 0 1 1\n",
      "37\t21\t0.0\t3\t0 2 7 12\n",
      "38\t30\t0.0\t3\t0 3 8 19\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_Feces_26_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_Feces_26_64_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_Feces_26_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_Feces_26_64_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_Feces_26_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_Feces_26_64_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_Feces_26_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_Feces_26_64_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:05       176,086 reads  @     32.3 µs/read;   1.86 M reads/minute\n",
      "Finished in 5.73 s (33 us/read; 1.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            176,086\n",
      "  Read 1 with adapter:                     193 (0.1%)\n",
      "  Read 2 with adapter:                      37 (0.0%)\n",
      "Pairs that were too short:                  12 (0.0%)\n",
      "Pairs written (passing filters):       176,074 (100.0%)\n",
      "\n",
      "Total basepairs processed:    88,395,172 bp\n",
      "  Read 1:    44,197,586 bp\n",
      "  Read 2:    44,197,586 bp\n",
      "Total written (filtered):     88,388,355 bp (100.0%)\n",
      "  Read 1:    44,193,984 bp\n",
      "  Read 2:    44,194,371 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 193 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t192\t2751.3\t0\t192\n",
      "14\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 37 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t2751.3\t0\t12\n",
      "5\t1\t172.0\t0\t1\n",
      "6\t9\t43.0\t0\t9\n",
      "36\t3\t0.0\t3\t0 0 1 2\n",
      "37\t7\t0.0\t3\t0 0 3 4\n",
      "38\t5\t0.0\t3\t0 0 2 3\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_Feces_30_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_Feces_30_58_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_Feces_30_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_Feces_30_58_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_Feces_30_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_Feces_30_58_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_Feces_30_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_Feces_30_58_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       198,221 reads  @     29.3 µs/read;   2.04 M reads/minute\n",
      "Finished in 5.86 s (30 us/read; 2.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            198,221\n",
      "  Read 1 with adapter:                     209 (0.1%)\n",
      "  Read 2 with adapter:                     281 (0.1%)\n",
      "Pairs that were too short:                  42 (0.0%)\n",
      "Pairs written (passing filters):       198,179 (100.0%)\n",
      "\n",
      "Total basepairs processed:    99,506,942 bp\n",
      "  Read 1:    49,753,471 bp\n",
      "  Read 2:    49,753,471 bp\n",
      "Total written (filtered):     99,484,247 bp (100.0%)\n",
      "  Read 1:    49,742,302 bp\n",
      "  Read 2:    49,741,945 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 209 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t209\t3097.2\t0\t209\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 281 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t227\t3097.2\t0\t227\n",
      "4\t2\t774.3\t0\t2\n",
      "5\t2\t193.6\t0\t2\n",
      "34\t1\t0.0\t3\t0 0 1\n",
      "35\t1\t0.0\t3\t0 0 0 1\n",
      "36\t6\t0.0\t3\t0 0 0 6\n",
      "37\t27\t0.0\t3\t0 3 7 17\n",
      "38\t14\t0.0\t3\t0 0 4 10\n",
      "39\t1\t0.0\t3\t0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_Feed_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_Feed_83_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_Feed_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_Feed_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_Feed_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_Feed_83_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_Feed_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_Feed_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       171,494 reads  @     33.2 µs/read;   1.81 M reads/minute\n",
      "Finished in 5.74 s (33 us/read; 1.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            171,494\n",
      "  Read 1 with adapter:                     146 (0.1%)\n",
      "  Read 2 with adapter:                      15 (0.0%)\n",
      "Pairs that were too short:                   4 (0.0%)\n",
      "Pairs written (passing filters):       171,490 (100.0%)\n",
      "\n",
      "Total basepairs processed:    86,089,988 bp\n",
      "  Read 1:    43,044,994 bp\n",
      "  Read 2:    43,044,994 bp\n",
      "Total written (filtered):     86,087,508 bp (100.0%)\n",
      "  Read 1:    43,043,551 bp\n",
      "  Read 2:    43,043,957 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 146 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t145\t2679.6\t0\t145\n",
      "4\t1\t669.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 15 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t2679.6\t0\t11\n",
      "37\t2\t0.0\t3\t0 0 0 2\n",
      "38\t2\t0.0\t3\t0 0 0 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_GUT_30_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_GUT_30_88_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_GUT_30_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_GUT_30_88_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_GUT_30_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_GUT_30_88_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_GUT_30_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_GUT_30_88_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       144,052 reads  @     27.0 µs/read;   2.22 M reads/minute\n",
      "Finished in 3.93 s (27 us/read; 2.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            144,052\n",
      "  Read 1 with adapter:                      15 (0.0%)\n",
      "  Read 2 with adapter:                      35 (0.0%)\n",
      "Pairs that were too short:                  20 (0.0%)\n",
      "Pairs written (passing filters):       144,032 (100.0%)\n",
      "\n",
      "Total basepairs processed:    72,314,104 bp\n",
      "  Read 1:    36,157,052 bp\n",
      "  Read 2:    36,157,052 bp\n",
      "Total written (filtered):     72,303,856 bp (100.0%)\n",
      "  Read 1:    36,151,987 bp\n",
      "  Read 2:    36,151,869 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 15 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t2250.8\t0\t15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 35 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t2250.8\t0\t11\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "36\t3\t0.0\t3\t0 0 0 3\n",
      "37\t7\t0.0\t3\t0 0 4 3\n",
      "38\t13\t0.0\t3\t0 1 2 10\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_SWTR_26_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_SWTR_26_53_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_SWTR_26_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_SWTR_26_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_SWTR_26_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_SWTR_26_53_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_SWTR_26_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_SWTR_26_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:09       297,299 reads  @     33.2 µs/read;   1.81 M reads/minute\n",
      "Finished in 9.92 s (33 us/read; 1.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            297,299\n",
      "  Read 1 with adapter:                     282 (0.1%)\n",
      "  Read 2 with adapter:                     232 (0.1%)\n",
      "Pairs that were too short:                 110 (0.0%)\n",
      "Pairs written (passing filters):       297,189 (100.0%)\n",
      "\n",
      "Total basepairs processed:   149,244,098 bp\n",
      "  Read 1:    74,622,049 bp\n",
      "  Read 2:    74,622,049 bp\n",
      "Total written (filtered):    149,187,006 bp (100.0%)\n",
      "  Read 1:    74,593,593 bp\n",
      "  Read 2:    74,593,413 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 282 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t282\t4645.3\t0\t282\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 232 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t101\t4645.3\t0\t101\n",
      "6\t1\t72.6\t0\t1\n",
      "35\t3\t0.0\t3\t0 0 0 3\n",
      "36\t17\t0.0\t3\t0 0 5 12\n",
      "37\t48\t0.0\t3\t0 4 12 32\n",
      "38\t60\t0.0\t3\t0 4 22 34\n",
      "39\t2\t0.0\t3\t0 0 0 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_SWTR_30_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_SWTR_30_49_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_SWTR_30_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_SWTR_30_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_SWTR_30_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR2_SWTR_30_49_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_SWTR_30_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR2_SWTR_30_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:07       243,383 reads  @     31.8 µs/read;   1.89 M reads/minute\n",
      "Finished in 7.81 s (32 us/read; 1.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            243,383\n",
      "  Read 1 with adapter:                     258 (0.1%)\n",
      "  Read 2 with adapter:                      98 (0.0%)\n",
      "Pairs that were too short:                  57 (0.0%)\n",
      "Pairs written (passing filters):       243,326 (100.0%)\n",
      "\n",
      "Total basepairs processed:   122,178,266 bp\n",
      "  Read 1:    61,089,133 bp\n",
      "  Read 2:    61,089,133 bp\n",
      "Total written (filtered):    122,148,464 bp (100.0%)\n",
      "  Read 1:    61,074,052 bp\n",
      "  Read 2:    61,074,412 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 258 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t258\t3802.9\t0\t258\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 98 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t3802.9\t0\t19\n",
      "4\t1\t950.7\t0\t1\n",
      "5\t13\t237.7\t0\t13\n",
      "36\t8\t0.0\t3\t0 0 3 5\n",
      "37\t21\t0.0\t3\t0 0 5 16\n",
      "38\t35\t0.0\t3\t0 2 6 27\n",
      "39\t1\t0.0\t3\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3A_GUT_26_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3A_GUT_26_91_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3A_GUT_26_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3A_GUT_26_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3A_GUT_26_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3A_GUT_26_91_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3A_GUT_26_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3A_GUT_26_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       229,380 reads  @     29.0 µs/read;   2.07 M reads/minute\n",
      "Finished in 6.69 s (29 us/read; 2.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            229,380\n",
      "  Read 1 with adapter:                     197 (0.1%)\n",
      "  Read 2 with adapter:                      59 (0.0%)\n",
      "Pairs that were too short:                  45 (0.0%)\n",
      "Pairs written (passing filters):       229,335 (100.0%)\n",
      "\n",
      "Total basepairs processed:   115,148,760 bp\n",
      "  Read 1:    57,574,380 bp\n",
      "  Read 2:    57,574,380 bp\n",
      "Total written (filtered):    115,125,410 bp (100.0%)\n",
      "  Read 1:    57,562,492 bp\n",
      "  Read 2:    57,562,918 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 197 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t196\t3584.1\t0\t196\n",
      "5\t1\t224.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 59 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t3584.1\t0\t8\n",
      "9\t1\t0.9\t0\t0 1\n",
      "13\t2\t0.0\t1\t0 2\n",
      "36\t3\t0.0\t3\t0 0 1 2\n",
      "37\t16\t0.0\t3\t0 1 3 12\n",
      "38\t27\t0.0\t3\t0 4 8 15\n",
      "39\t2\t0.0\t3\t0 0 0 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_CF_26_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_CF_26_73_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_CF_26_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_CF_26_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_CF_26_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_CF_26_73_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_CF_26_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_CF_26_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       142,275 reads  @     32.0 µs/read;   1.88 M reads/minute\n",
      "Finished in 4.59 s (32 us/read; 1.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            142,275\n",
      "  Read 1 with adapter:                      60 (0.0%)\n",
      "  Read 2 with adapter:                      23 (0.0%)\n",
      "Pairs that were too short:                   2 (0.0%)\n",
      "Pairs written (passing filters):       142,273 (100.0%)\n",
      "\n",
      "Total basepairs processed:    71,422,050 bp\n",
      "  Read 1:    35,711,025 bp\n",
      "  Read 2:    35,711,025 bp\n",
      "Total written (filtered):     71,420,735 bp (100.0%)\n",
      "  Read 1:    35,710,349 bp\n",
      "  Read 2:    35,710,386 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 60 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t2223.0\t0\t58\n",
      "149\t2\t0.0\t6\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 23 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t2223.0\t0\t15\n",
      "4\t1\t555.8\t0\t1\n",
      "5\t2\t138.9\t0\t2\n",
      "6\t1\t34.7\t0\t1\n",
      "36\t2\t0.0\t3\t0 0 0 2\n",
      "149\t2\t0.0\t6\t0 0 1 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_CF_30_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_CF_30_68_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_CF_30_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_CF_30_68_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_CF_30_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_CF_30_68_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_CF_30_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_CF_30_68_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        68,135 reads  @     41.1 µs/read;   1.46 M reads/minute\n",
      "Finished in 2.85 s (42 us/read; 1.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,135\n",
      "  Read 1 with adapter:                      43 (0.1%)\n",
      "  Read 2 with adapter:                      22 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        68,135 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,203,770 bp\n",
      "  Read 1:    17,101,885 bp\n",
      "  Read 2:    17,101,885 bp\n",
      "Total written (filtered):     34,203,564 bp (100.0%)\n",
      "  Read 1:    17,101,745 bp\n",
      "  Read 2:    17,101,819 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 43 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t1064.6\t0\t42\n",
      "14\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 22 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t1064.6\t0\t22\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_DP_26_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_DP_26_79_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_DP_26_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_DP_26_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_DP_26_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_DP_26_79_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_DP_26_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_DP_26_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       129,170 reads  @     28.8 µs/read;   2.08 M reads/minute\n",
      "Finished in 3.76 s (29 us/read; 2.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            129,170\n",
      "  Read 1 with adapter:                      70 (0.1%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       129,169 (100.0%)\n",
      "\n",
      "Total basepairs processed:    64,843,340 bp\n",
      "  Read 1:    32,421,670 bp\n",
      "  Read 2:    32,421,670 bp\n",
      "Total written (filtered):     64,842,570 bp (100.0%)\n",
      "  Read 1:    32,421,209 bp\n",
      "  Read 2:    32,421,361 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 70 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t70\t2018.3\t0\t70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 9 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t2018.3\t0\t6\n",
      "4\t1\t504.6\t0\t1\n",
      "36\t1\t0.0\t3\t0 0 0 1\n",
      "38\t1\t0.0\t3\t0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_DP_30_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_DP_30_76_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_DP_30_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_DP_30_76_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_DP_30_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_DP_30_76_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_DP_30_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_DP_30_76_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:06       228,931 reads  @     28.9 µs/read;   2.08 M reads/minute\n",
      "Finished in 6.65 s (29 us/read; 2.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            228,931\n",
      "  Read 1 with adapter:                     185 (0.1%)\n",
      "  Read 2 with adapter:                     198 (0.1%)\n",
      "Pairs that were too short:                   8 (0.0%)\n",
      "Pairs written (passing filters):       228,923 (100.0%)\n",
      "\n",
      "Total basepairs processed:   114,923,362 bp\n",
      "  Read 1:    57,461,681 bp\n",
      "  Read 2:    57,461,681 bp\n",
      "Total written (filtered):    114,917,679 bp (100.0%)\n",
      "  Read 1:    57,459,118 bp\n",
      "  Read 2:    57,458,561 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 185 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t185\t3577.0\t0\t185\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 198 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t3577.0\t0\t17\n",
      "4\t2\t894.3\t0\t2\n",
      "5\t3\t223.6\t0\t3\n",
      "6\t167\t55.9\t0\t167\n",
      "36\t1\t0.0\t3\t0 0 0 1\n",
      "37\t3\t0.0\t3\t0 0 1 2\n",
      "38\t5\t0.0\t3\t0 0 2 3\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_Feces_26_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_Feces_26_63_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_Feces_26_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_Feces_26_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_Feces_26_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_Feces_26_63_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_Feces_26_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_Feces_26_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:04       161,549 reads  @     29.0 µs/read;   2.07 M reads/minute\n",
      "Finished in 4.73 s (29 us/read; 2.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            161,549\n",
      "  Read 1 with adapter:                     188 (0.1%)\n",
      "  Read 2 with adapter:                      82 (0.1%)\n",
      "Pairs that were too short:                  39 (0.0%)\n",
      "Pairs written (passing filters):       161,510 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,097,598 bp\n",
      "  Read 1:    40,548,799 bp\n",
      "  Read 2:    40,548,799 bp\n",
      "Total written (filtered):     81,076,929 bp (100.0%)\n",
      "  Read 1:    40,538,446 bp\n",
      "  Read 2:    40,538,483 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 188 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t188\t2524.2\t0\t188\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 82 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t2524.2\t0\t30\n",
      "6\t1\t39.4\t0\t1\n",
      "35\t1\t0.0\t3\t0 0 0 1\n",
      "36\t11\t0.0\t3\t0 0 3 8\n",
      "37\t24\t0.0\t3\t0 0 4 20\n",
      "38\t15\t0.0\t3\t0 0 4 11\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_Feces_30_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_Feces_30_57_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_Feces_30_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_Feces_30_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_Feces_30_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_Feces_30_57_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_Feces_30_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_Feces_30_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       191,890 reads  @     30.4 µs/read;   1.97 M reads/minute\n",
      "Finished in 5.87 s (31 us/read; 1.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            191,890\n",
      "  Read 1 with adapter:                     200 (0.1%)\n",
      "  Read 2 with adapter:                      40 (0.0%)\n",
      "Pairs that were too short:                  13 (0.0%)\n",
      "Pairs written (passing filters):       191,877 (100.0%)\n",
      "\n",
      "Total basepairs processed:    96,328,780 bp\n",
      "  Read 1:    48,164,390 bp\n",
      "  Read 2:    48,164,390 bp\n",
      "Total written (filtered):     96,321,396 bp (100.0%)\n",
      "  Read 1:    48,160,527 bp\n",
      "  Read 2:    48,160,869 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 200 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t200\t2998.3\t0\t200\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 40 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t2998.3\t0\t16\n",
      "5\t6\t187.4\t0\t6\n",
      "36\t5\t0.0\t3\t0 0 1 4\n",
      "37\t9\t0.0\t3\t0 0 0 9\n",
      "38\t4\t0.0\t3\t0 0 1 3\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_Feed_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_Feed_82_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_Feed_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_Feed_82_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_Feed_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_Feed_82_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_Feed_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_Feed_82_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:04       161,140 reads  @     27.3 µs/read;   2.20 M reads/minute\n",
      "Finished in 4.45 s (28 us/read; 2.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            161,140\n",
      "  Read 1 with adapter:                     177 (0.1%)\n",
      "  Read 2 with adapter:                      25 (0.0%)\n",
      "Pairs that were too short:                   4 (0.0%)\n",
      "Pairs written (passing filters):       161,136 (100.0%)\n",
      "\n",
      "Total basepairs processed:    80,892,280 bp\n",
      "  Read 1:    40,446,140 bp\n",
      "  Read 2:    40,446,140 bp\n",
      "Total written (filtered):     80,889,660 bp (100.0%)\n",
      "  Read 1:    40,444,605 bp\n",
      "  Read 2:    40,445,055 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 177 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t177\t2517.8\t0\t177\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 25 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t2517.8\t0\t12\n",
      "4\t2\t629.5\t0\t2\n",
      "5\t5\t157.4\t0\t5\n",
      "6\t2\t39.3\t0\t2\n",
      "38\t3\t0.0\t3\t0 0 1 2\n",
      "39\t1\t0.0\t3\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_GUT_30_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_GUT_30_87_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_GUT_30_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_GUT_30_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_GUT_30_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_GUT_30_87_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_GUT_30_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_GUT_30_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:04       163,851 reads  @     26.2 µs/read;   2.29 M reads/minute\n",
      "Finished in 4.34 s (26 us/read; 2.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            163,851\n",
      "  Read 1 with adapter:                       6 (0.0%)\n",
      "  Read 2 with adapter:                     334 (0.2%)\n",
      "Pairs that were too short:                   4 (0.0%)\n",
      "Pairs written (passing filters):       163,847 (100.0%)\n",
      "\n",
      "Total basepairs processed:    82,253,202 bp\n",
      "  Read 1:    41,126,601 bp\n",
      "  Read 2:    41,126,601 bp\n",
      "Total written (filtered):     82,250,010 bp (100.0%)\n",
      "  Read 1:    41,125,579 bp\n",
      "  Read 2:    41,124,431 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 6 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t2560.2\t0\t6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 334 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t319\t2560.2\t0\t319\n",
      "5\t6\t160.0\t0\t6\n",
      "35\t1\t0.0\t3\t0 1\n",
      "36\t4\t0.0\t3\t0 0 1 3\n",
      "37\t1\t0.0\t3\t0 0 0 1\n",
      "38\t3\t0.0\t3\t0 0 1 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_SWTR_26_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_SWTR_26_52_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_SWTR_26_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_SWTR_26_52_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_SWTR_26_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_SWTR_26_52_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_SWTR_26_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_SWTR_26_52_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:07       258,947 reads  @     29.9 µs/read;   2.01 M reads/minute\n",
      "Finished in 7.80 s (30 us/read; 1.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            258,947\n",
      "  Read 1 with adapter:                     261 (0.1%)\n",
      "  Read 2 with adapter:                     145 (0.1%)\n",
      "Pairs that were too short:                  56 (0.0%)\n",
      "Pairs written (passing filters):       258,891 (100.0%)\n",
      "\n",
      "Total basepairs processed:   129,991,394 bp\n",
      "  Read 1:    64,995,697 bp\n",
      "  Read 2:    64,995,697 bp\n",
      "Total written (filtered):    129,961,851 bp (100.0%)\n",
      "  Read 1:    64,980,858 bp\n",
      "  Read 2:    64,980,993 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 261 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t261\t4046.0\t0\t261\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 145 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t76\t4046.0\t0\t76\n",
      "4\t1\t1011.5\t0\t1\n",
      "21\t1\t0.0\t2\t0 1\n",
      "35\t1\t0.0\t3\t0 0 0 1\n",
      "36\t10\t0.0\t3\t0 0 2 8\n",
      "37\t31\t0.0\t3\t0 1 3 27\n",
      "38\t24\t0.0\t3\t0 1 5 18\n",
      "39\t1\t0.0\t3\t0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_SWTR_30_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_SWTR_30_48_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_SWTR_30_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_SWTR_30_48_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_SWTR_30_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR3_SWTR_30_48_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_SWTR_30_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR3_SWTR_30_48_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:05       146,902 reads  @     34.8 µs/read;   1.72 M reads/minute\n",
      "Finished in 5.16 s (35 us/read; 1.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            146,902\n",
      "  Read 1 with adapter:                      65 (0.0%)\n",
      "  Read 2 with adapter:                      53 (0.0%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       146,901 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,744,804 bp\n",
      "  Read 1:    36,872,402 bp\n",
      "  Read 2:    36,872,402 bp\n",
      "Total written (filtered):     73,743,912 bp (100.0%)\n",
      "  Read 1:    36,871,959 bp\n",
      "  Read 2:    36,871,953 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 65 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t64\t2295.3\t0\t64\n",
      "139\t1\t0.0\t6\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 53 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t38\t2295.3\t0\t38\n",
      "4\t3\t573.8\t0\t3\n",
      "6\t12\t35.9\t0\t12\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4A_GUT_26_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4A_GUT_26_90_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4A_GUT_26_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4A_GUT_26_90_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4A_GUT_26_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4A_GUT_26_90_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4A_GUT_26_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4A_GUT_26_90_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       141,180 reads  @     27.6 µs/read;   2.17 M reads/minute\n",
      "Finished in 3.94 s (28 us/read; 2.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            141,180\n",
      "  Read 1 with adapter:                      10 (0.0%)\n",
      "  Read 2 with adapter:                      22 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       141,180 (100.0%)\n",
      "\n",
      "Total basepairs processed:    70,872,360 bp\n",
      "  Read 1:    35,436,180 bp\n",
      "  Read 2:    35,436,180 bp\n",
      "Total written (filtered):     70,872,264 bp (100.0%)\n",
      "  Read 1:    35,436,150 bp\n",
      "  Read 2:    35,436,114 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 10 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t2205.9\t0\t10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 22 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t2205.9\t0\t22\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_CF_26_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_CF_26_72_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_CF_26_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_CF_26_72_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_CF_26_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_CF_26_72_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_CF_26_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_CF_26_72_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:05       161,323 reads  @     32.3 µs/read;   1.86 M reads/minute\n",
      "Finished in 5.26 s (33 us/read; 1.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            161,323\n",
      "  Read 1 with adapter:                      61 (0.0%)\n",
      "  Read 2 with adapter:                      17 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       161,323 (100.0%)\n",
      "\n",
      "Total basepairs processed:    80,984,146 bp\n",
      "  Read 1:    40,492,073 bp\n",
      "  Read 2:    40,492,073 bp\n",
      "Total written (filtered):     80,983,889 bp (100.0%)\n",
      "  Read 1:    40,491,890 bp\n",
      "  Read 2:    40,491,999 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 61 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t61\t2520.7\t0\t61\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 17 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t2520.7\t0\t12\n",
      "4\t3\t630.2\t0\t3\n",
      "6\t1\t39.4\t0\t1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_CF_30_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_CF_30_67_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_CF_30_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_CF_30_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_CF_30_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_CF_30_67_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_CF_30_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_CF_30_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        80,865 reads  @     37.4 µs/read;   1.61 M reads/minute\n",
      "Finished in 3.08 s (38 us/read; 1.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,865\n",
      "  Read 1 with adapter:                      34 (0.0%)\n",
      "  Read 2 with adapter:                       9 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        80,865 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,594,230 bp\n",
      "  Read 1:    20,297,115 bp\n",
      "  Read 2:    20,297,115 bp\n",
      "Total written (filtered):     40,594,100 bp (100.0%)\n",
      "  Read 1:    20,297,013 bp\n",
      "  Read 2:    20,297,087 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 34 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t1263.5\t0\t34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 9 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1263.5\t0\t8\n",
      "4\t1\t315.9\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_Feces_26_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_Feces_26_62_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_Feces_26_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_Feces_26_62_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_Feces_26_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_Feces_26_62_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_Feces_26_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_Feces_26_62_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:05       172,824 reads  @     32.4 µs/read;   1.85 M reads/minute\n",
      "Finished in 5.66 s (33 us/read; 1.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            172,824\n",
      "  Read 1 with adapter:                     171 (0.1%)\n",
      "  Read 2 with adapter:                      76 (0.0%)\n",
      "Pairs that were too short:                  53 (0.0%)\n",
      "Pairs written (passing filters):       172,771 (100.0%)\n",
      "\n",
      "Total basepairs processed:    86,757,648 bp\n",
      "  Read 1:    43,378,824 bp\n",
      "  Read 2:    43,378,824 bp\n",
      "Total written (filtered):     86,729,934 bp (100.0%)\n",
      "  Read 1:    43,365,008 bp\n",
      "  Read 2:    43,364,926 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 171 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t171\t2700.4\t0\t171\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 76 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t2700.4\t0\t7\n",
      "35\t2\t0.0\t3\t0 0 0 2\n",
      "36\t14\t0.0\t3\t0 0 2 12\n",
      "37\t33\t0.0\t3\t0 2 12 19\n",
      "38\t18\t0.0\t3\t0 1 5 12\n",
      "39\t2\t0.0\t3\t0 0 1 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_Feces_30_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_Feces_30_56_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_Feces_30_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_Feces_30_56_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_Feces_30_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_Feces_30_56_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_Feces_30_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_Feces_30_56_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8=--------] 00:00:06       197,245 reads  @     31.8 µs/read;   1.89 M reads/minute\n",
      "Finished in 6.31 s (32 us/read; 1.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            197,245\n",
      "  Read 1 with adapter:                     179 (0.1%)\n",
      "  Read 2 with adapter:                      31 (0.0%)\n",
      "Pairs that were too short:                  12 (0.0%)\n",
      "Pairs written (passing filters):       197,233 (100.0%)\n",
      "\n",
      "Total basepairs processed:    99,016,990 bp\n",
      "  Read 1:    49,508,495 bp\n",
      "  Read 2:    49,508,495 bp\n",
      "Total written (filtered):     99,010,230 bp (100.0%)\n",
      "  Read 1:    49,504,946 bp\n",
      "  Read 2:    49,505,284 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 179 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t179\t3082.0\t0\t179\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 31 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t3082.0\t0\t12\n",
      "4\t1\t770.5\t0\t1\n",
      "5\t1\t192.6\t0\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "35\t3\t0.0\t3\t0 0 0 3\n",
      "36\t1\t0.0\t3\t0 0 0 1\n",
      "37\t6\t0.0\t3\t0 0 0 6\n",
      "38\t6\t0.0\t3\t0 0 2 4\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_GUT_30_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_GUT_30_86_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_GUT_30_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_GUT_30_86_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_GUT_30_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_GUT_30_86_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_GUT_30_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_GUT_30_86_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       127,480 reads  @     27.6 µs/read;   2.18 M reads/minute\n",
      "Finished in 3.55 s (28 us/read; 2.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            127,480\n",
      "  Read 1 with adapter:                       5 (0.0%)\n",
      "  Read 2 with adapter:                      13 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       127,480 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,994,960 bp\n",
      "  Read 1:    31,997,480 bp\n",
      "  Read 2:    31,997,480 bp\n",
      "Total written (filtered):     63,994,904 bp (100.0%)\n",
      "  Read 1:    31,997,465 bp\n",
      "  Read 2:    31,997,439 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 5 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1991.9\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 13 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t1991.9\t0\t12\n",
      "5\t1\t124.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_SWTR_26_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_SWTR_26_51_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_SWTR_26_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_SWTR_26_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_SWTR_26_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_SWTR_26_51_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_SWTR_26_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_SWTR_26_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[   8<-------] 00:00:07       246,807 reads  @     31.1 µs/read;   1.93 M reads/minute\n",
      "Finished in 7.73 s (31 us/read; 1.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            246,807\n",
      "  Read 1 with adapter:                     309 (0.1%)\n",
      "  Read 2 with adapter:                      99 (0.0%)\n",
      "Pairs that were too short:                  77 (0.0%)\n",
      "Pairs written (passing filters):       246,730 (100.0%)\n",
      "\n",
      "Total basepairs processed:   123,897,114 bp\n",
      "  Read 1:    61,948,557 bp\n",
      "  Read 2:    61,948,557 bp\n",
      "Total written (filtered):    123,857,134 bp (100.0%)\n",
      "  Read 1:    61,928,303 bp\n",
      "  Read 2:    61,928,831 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 309 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t309\t3856.4\t0\t309\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 99 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t3856.4\t0\t10\n",
      "5\t2\t241.0\t0\t2\n",
      "35\t1\t0.0\t3\t0 0 0 1\n",
      "36\t9\t0.0\t3\t0 0 4 5\n",
      "37\t34\t0.0\t3\t0 2 9 23\n",
      "38\t43\t0.0\t3\t0 4 10 29\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_SWTR_30_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_SWTR_30_47_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_SWTR_30_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_SWTR_30_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_SWTR_30_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR4_SWTR_30_47_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_SWTR_30_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR4_SWTR_30_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       120,568 reads  @     35.5 µs/read;   1.69 M reads/minute\n",
      "Finished in 4.33 s (36 us/read; 1.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            120,568\n",
      "  Read 1 with adapter:                      89 (0.1%)\n",
      "  Read 2 with adapter:                     102 (0.1%)\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):       120,567 (100.0%)\n",
      "\n",
      "Total basepairs processed:    60,525,136 bp\n",
      "  Read 1:    30,262,568 bp\n",
      "  Read 2:    30,262,568 bp\n",
      "Total written (filtered):     60,524,058 bp (100.0%)\n",
      "  Read 1:    30,262,049 bp\n",
      "  Read 2:    30,262,009 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 89 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t88\t1883.9\t0\t88\n",
      "4\t1\t471.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 102 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t98\t1883.9\t0\t98\n",
      "4\t1\t471.0\t0\t1\n",
      "5\t2\t117.7\t0\t2\n",
      "38\t1\t0.0\t3\t0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_CF_26_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_CF_26_71_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_CF_26_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_CF_26_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_CF_26_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_CF_26_71_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_CF_26_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_CF_26_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        90,116 reads  @     39.4 µs/read;   1.52 M reads/minute\n",
      "Finished in 3.60 s (40 us/read; 1.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             90,116\n",
      "  Read 1 with adapter:                      39 (0.0%)\n",
      "  Read 2 with adapter:                      22 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        90,116 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,238,232 bp\n",
      "  Read 1:    22,619,116 bp\n",
      "  Read 2:    22,619,116 bp\n",
      "Total written (filtered):     45,238,047 bp (100.0%)\n",
      "  Read 1:    22,618,999 bp\n",
      "  Read 2:    22,619,048 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 39 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t1408.1\t0\t39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 22 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t1408.1\t0\t21\n",
      "5\t1\t88.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_CF_30_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_CF_30_66_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_CF_30_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_CF_30_66_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_CF_30_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_CF_30_66_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_CF_30_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_CF_30_66_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:03       105,507 reads  @     32.5 µs/read;   1.85 M reads/minute\n",
      "Finished in 3.48 s (33 us/read; 1.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            105,507\n",
      "  Read 1 with adapter:                      52 (0.0%)\n",
      "  Read 2 with adapter:                      31 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       105,507 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,964,514 bp\n",
      "  Read 1:    26,482,257 bp\n",
      "  Read 2:    26,482,257 bp\n",
      "Total written (filtered):     52,964,231 bp (100.0%)\n",
      "  Read 1:    26,482,101 bp\n",
      "  Read 2:    26,482,130 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 52 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t52\t1648.5\t0\t52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 31 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t1648.5\t0\t29\n",
      "4\t1\t412.1\t0\t1\n",
      "36\t1\t0.0\t3\t0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_Feces_26_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_Feces_26_61_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_Feces_26_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_Feces_26_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_Feces_26_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_Feces_26_61_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_Feces_26_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_Feces_26_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:06       196,543 reads  @     30.5 µs/read;   1.96 M reads/minute\n",
      "Finished in 6.05 s (31 us/read; 1.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            196,543\n",
      "  Read 1 with adapter:                     194 (0.1%)\n",
      "  Read 2 with adapter:                     194 (0.1%)\n",
      "Pairs that were too short:                  10 (0.0%)\n",
      "Pairs written (passing filters):       196,533 (100.0%)\n",
      "\n",
      "Total basepairs processed:    98,664,586 bp\n",
      "  Read 1:    49,332,293 bp\n",
      "  Read 2:    49,332,293 bp\n",
      "Total written (filtered):     98,658,345 bp (100.0%)\n",
      "  Read 1:    49,329,201 bp\n",
      "  Read 2:    49,329,144 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 194 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t194\t3071.0\t0\t194\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 194 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t154\t3071.0\t0\t154\n",
      "4\t22\t767.7\t0\t22\n",
      "6\t1\t48.0\t0\t1\n",
      "8\t6\t3.0\t0\t6\n",
      "35\t1\t0.0\t3\t0 0 1\n",
      "37\t6\t0.0\t3\t0 0 2 4\n",
      "38\t3\t0.0\t3\t0 0 1 2\n",
      "39\t1\t0.0\t3\t0 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_Feces_30_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_Feces_30_55_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_Feces_30_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_Feces_30_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_Feces_30_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_Feces_30_55_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_Feces_30_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_Feces_30_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[  8<--------] 00:00:06       209,154 reads  @     32.1 µs/read;   1.87 M reads/minute\n",
      "Finished in 6.77 s (32 us/read; 1.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            209,154\n",
      "  Read 1 with adapter:                     205 (0.1%)\n",
      "  Read 2 with adapter:                     184 (0.1%)\n",
      "Pairs that were too short:                  10 (0.0%)\n",
      "Pairs written (passing filters):       209,144 (100.0%)\n",
      "\n",
      "Total basepairs processed:   104,995,308 bp\n",
      "  Read 1:    52,497,654 bp\n",
      "  Read 2:    52,497,654 bp\n",
      "Total written (filtered):    104,989,029 bp (100.0%)\n",
      "  Read 1:    52,494,529 bp\n",
      "  Read 2:    52,494,500 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 205 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t205\t3268.0\t0\t205\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 184 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t73\t3268.0\t0\t73\n",
      "4\t90\t817.0\t0\t90\n",
      "5\t1\t204.3\t0\t1\n",
      "6\t10\t51.1\t0\t10\n",
      "37\t3\t0.0\t3\t0 0 1 2\n",
      "38\t7\t0.0\t3\t0 0 1 6\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_GUT_30_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_GUT_30_85_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_GUT_30_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_GUT_30_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_GUT_30_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR5_GUT_30_85_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_GUT_30_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR5_GUT_30_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8=---------] 00:00:04       168,524 reads  @     25.5 µs/read;   2.35 M reads/minute\n",
      "Finished in 4.34 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            168,524\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                      54 (0.0%)\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       168,524 (100.0%)\n",
      "\n",
      "Total basepairs processed:    84,599,048 bp\n",
      "  Read 1:    42,299,524 bp\n",
      "  Read 2:    42,299,524 bp\n",
      "Total written (filtered):     84,598,868 bp (100.0%)\n",
      "  Read 1:    42,299,524 bp\n",
      "  Read 2:    42,299,344 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 0 times.\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 54 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t49\t2633.2\t0\t49\n",
      "5\t1\t164.6\t0\t1\n",
      "7\t4\t10.3\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR6_Feces_26_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR6_Feces_26_60_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR6_Feces_26_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR6_Feces_26_60_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.6 with Python 3.6.7\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 215 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR6_Feces_26_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-8o5jmtbr/UR6_Feces_26_60_L001_R2_001.fastq.gz --front AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA -G CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR6_Feces_26_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-lp6rqbc7/a8eaf4d9-4a7f-42ec-ba0d-053d170de06d/data/UR6_Feces_26_60_L001_R2_001.fastq.gz\n",
      "Processing reads on 16 cores in paired-end mode ...\n",
      "[ 8<---------] 00:00:03       135,139 reads  @     28.3 µs/read;   2.12 M reads/minute\n",
      "Finished in 3.87 s (29 us/read; 2.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            135,139\n",
      "  Read 1 with adapter:                      20 (0.0%)\n",
      "  Read 2 with adapter:                      39 (0.0%)\n",
      "Pairs that were too short:                  18 (0.0%)\n",
      "Pairs written (passing filters):       135,121 (100.0%)\n",
      "\n",
      "Total basepairs processed:    67,839,778 bp\n",
      "  Read 1:    33,919,889 bp\n",
      "  Read 2:    33,919,889 bp\n",
      "Total written (filtered):     67,830,572 bp (100.0%)\n",
      "  Read 1:    33,915,310 bp\n",
      "  Read 2:    33,915,262 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 60; Trimmed: 20 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-59 bp: 5; 60 bp: 6\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t2111.5\t0\t19\n",
      "4\t1\t527.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 61; Trimmed: 39 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-29 bp: 2; 30-39 bp: 3; 40-49 bp: 4; 50-55 bp: 5\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t2111.5\t0\t15\n",
      "5\t1\t132.0\t0\t1\n",
      "6\t4\t33.0\t0\t4\n",
      "35\t1\t0.0\t3\t0 0 0 1\n",
      "37\t6\t0.0\t3\t0 0 2 4\n",
      "38\t11\t0.0\t3\t0 0 3 8\n",
      "39\t1\t0.0\t3\t0 0 0 1\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: paired-end-trimmed-demux.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime cutadapt trim-paired \\\n",
    "--i-demultiplexed-sequences BioProjectPRJNA376395/paired-end-demux.qza \\\n",
    "--o-trimmed-sequences paired-end-trimmed-demux.qza \\\n",
    "--p-cores 16 --p-front-f AATGATACGGCGACCACCGAGATCTACACTATGGTAATTGTGTGCCAGCMGCCGCGGTAA \\\n",
    "--p-front-r CAAGAGAAGACGGCATACGAGATNNNNNNAGTCAGTCAGCCGGACTACHVGGGTWTCTAAT \\\n",
    "--p-minimum-length 215 \\\n",
    "--verbose "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: paired-end-trimmed-demux.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime demux summarize \\\n",
    "--i-data paired-end-trimmed-demux.qza \\\n",
    "--o-visualization paired-end-trimmed-demux.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-kuazgh6e')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 740912b9-ee65-4bf9-915b-b323b3145964>"
      ]
     },
     "execution_count": 88,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load (\"paired-end-trimmed-demux.qzv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Usage: \u001b[34mqiime dada2 denoise-paired\u001b[0m [OPTIONS]\n",
      "\n",
      "  This method denoises paired-end sequences, dereplicates them, and filters\n",
      "  chimeras.\n",
      "\n",
      "\u001b[1mInputs\u001b[0m:\n",
      "  \u001b[34m\u001b[4m--i-demultiplexed-seqs\u001b[0m ARTIFACT \u001b[32mSampleData[PairedEndSequencesWithQuality]\u001b[0m\n",
      "                         The paired-end demultiplexed sequences to be\n",
      "                         denoised.                                  \u001b[35m[required]\u001b[0m\n",
      "\u001b[1mParameters\u001b[0m:\n",
      "  \u001b[34m\u001b[4m--p-trunc-len-f\u001b[0m INTEGER\n",
      "                         Position at which forward read sequences should be\n",
      "                         truncated due to decrease in quality. This truncates\n",
      "                         the 3' end of the of the input sequences, which will\n",
      "                         be the bases that were sequenced in the last cycles.\n",
      "                         Reads that are shorter than this value will be\n",
      "                         discarded. After this parameter is applied there must\n",
      "                         still be at least a 20 nucleotide overlap between the\n",
      "                         forward and reverse reads. If 0 is provided, no\n",
      "                         truncation or length filtering will be performed\n",
      "                                                                    \u001b[35m[required]\u001b[0m\n",
      "  \u001b[34m\u001b[4m--p-trunc-len-r\u001b[0m INTEGER\n",
      "                         Position at which reverse read sequences should be\n",
      "                         truncated due to decrease in quality. This truncates\n",
      "                         the 3' end of the of the input sequences, which will\n",
      "                         be the bases that were sequenced in the last cycles.\n",
      "                         Reads that are shorter than this value will be\n",
      "                         discarded. After this parameter is applied there must\n",
      "                         still be at least a 20 nucleotide overlap between the\n",
      "                         forward and reverse reads. If 0 is provided, no\n",
      "                         truncation or length filtering will be performed\n",
      "                                                                    \u001b[35m[required]\u001b[0m\n",
      "  \u001b[34m--p-trim-left-f\u001b[0m INTEGER\n",
      "                         Position at which forward read sequences should be\n",
      "                         trimmed due to low quality. This trims the 5' end of\n",
      "                         the input sequences, which will be the bases that\n",
      "                         were sequenced in the first cycles.      \u001b[35m[default: 0]\u001b[0m\n",
      "  \u001b[34m--p-trim-left-r\u001b[0m INTEGER\n",
      "                         Position at which reverse read sequences should be\n",
      "                         trimmed due to low quality. This trims the 5' end of\n",
      "                         the input sequences, which will be the bases that\n",
      "                         were sequenced in the first cycles.      \u001b[35m[default: 0]\u001b[0m\n",
      "  \u001b[34m--p-max-ee-f\u001b[0m NUMBER    Forward reads with number of expected errors higher\n",
      "                         than this value will be discarded.     \u001b[35m[default: 2.0]\u001b[0m\n",
      "  \u001b[34m--p-max-ee-r\u001b[0m NUMBER    Reverse reads with number of expected errors higher\n",
      "                         than this value will be discarded.     \u001b[35m[default: 2.0]\u001b[0m\n",
      "  \u001b[34m--p-trunc-q\u001b[0m INTEGER    Reads are truncated at the first instance of a\n",
      "                         quality score less than or equal to this value. If\n",
      "                         the resulting read is then shorter than `\u001b[4mtrunc-len-f\u001b[0m`\n",
      "                         or `\u001b[4mtrunc-len-r\u001b[0m` (depending on the direction of the\n",
      "                         read) it is discarded.                   \u001b[35m[default: 2]\u001b[0m\n",
      "  \u001b[34m--p-chimera-method\u001b[0m TEXT \u001b[32mChoices('consensus', 'pooled', 'none')\u001b[0m\n",
      "                         The method used to remove chimeras. \"none\": No\n",
      "                         chimera removal is performed. \"pooled\": All reads are\n",
      "                         pooled prior to chimera detection. \"consensus\":\n",
      "                         Chimeras are detected in samples individually, and\n",
      "                         sequences found chimeric in a sufficient fraction of\n",
      "                         samples are removed.           \u001b[35m[default: 'consensus']\u001b[0m\n",
      "  \u001b[34m--p-min-fold-parent-over-abundance\u001b[0m NUMBER\n",
      "                         The minimum abundance of potential parents of a\n",
      "                         sequence being tested as chimeric, expressed as a\n",
      "                         fold-change versus the abundance of the sequence\n",
      "                         being tested. Values should be greater than or equal\n",
      "                         to 1 (i.e. parents should be more abundant than the\n",
      "                         sequence being tested). This parameter has no effect\n",
      "                         if \u001b[4mchimera-method\u001b[0m is \"none\".           \u001b[35m[default: 1.0]\u001b[0m\n",
      "  \u001b[34m--p-n-threads\u001b[0m INTEGER  The number of threads to use for multithreaded\n",
      "                         processing. If 0 is provided, all available cores\n",
      "                         will be used.                            \u001b[35m[default: 1]\u001b[0m\n",
      "  \u001b[34m--p-n-reads-learn\u001b[0m INTEGER\n",
      "                         The number of reads to use when training the error\n",
      "                         model. Smaller numbers will result in a shorter run\n",
      "                         time but a less reliable error model.\n",
      "                                                            \u001b[35m[default: 1000000]\u001b[0m\n",
      "  \u001b[34m--p-hashed-feature-ids\u001b[0m / \u001b[34m--p-no-hashed-feature-ids\u001b[0m\n",
      "                         If true, the feature ids in the resulting table will\n",
      "                         be presented as hashes of the sequences defining each\n",
      "                         feature. The hash will always be the same for the\n",
      "                         same sequence so this allows feature tables to be\n",
      "                         merged across runs of this method. You should only\n",
      "                         merge tables if the exact same parameters are used\n",
      "                         for each run.                         \u001b[35m[default: True]\u001b[0m\n",
      "\u001b[1mOutputs\u001b[0m:\n",
      "  \u001b[34m\u001b[4m--o-table\u001b[0m ARTIFACT \u001b[32mFeatureTable[Frequency]\u001b[0m\n",
      "                         The resulting feature table.               \u001b[35m[required]\u001b[0m\n",
      "  \u001b[34m\u001b[4m--o-representative-sequences\u001b[0m ARTIFACT \u001b[32mFeatureData[Sequence]\u001b[0m\n",
      "                         The resulting feature sequences. Each feature in the\n",
      "                         feature table will be represented by exactly one\n",
      "                         sequence, and these sequences will be the joined\n",
      "                         paired-end sequences.                      \u001b[35m[required]\u001b[0m\n",
      "  \u001b[34m\u001b[4m--o-denoising-stats\u001b[0m ARTIFACT \u001b[32mSampleData[DADA2Stats]\u001b[0m\n",
      "                                                                    \u001b[35m[required]\u001b[0m\n",
      "\u001b[1mMiscellaneous\u001b[0m:\n",
      "  \u001b[34m--output-dir\u001b[0m PATH      Output unspecified results to a directory\n",
      "  \u001b[34m--verbose\u001b[0m / \u001b[34m--quiet\u001b[0m    Display verbose output to stdout and/or stderr\n",
      "                         during execution of this action. Or silence output if\n",
      "                         execution is successful (silence is golden).\n",
      "  \u001b[34m--citations\u001b[0m            Show citations and exit.\n",
      "  \u001b[34m--help\u001b[0m                 Show this message and exit.\n",
      "\n",
      "\u001b[33m                  There were some problems with the command:                  \u001b[0m\n",
      "\u001b[31m\u001b[1m (1/7) Missing option \"--i-demultiplexed-seqs\".\u001b[0m\n",
      "\u001b[31m\u001b[1m (2/7) Missing option \"--p-trunc-len-f\".\u001b[0m\n",
      "\u001b[31m\u001b[1m (3/7) Missing option \"--p-trunc-len-r\".\u001b[0m\n",
      "\u001b[31m\u001b[1m (4/7) Missing option \"--o-table\".  (\"--output-dir\" may also be used)\u001b[0m\n",
      "\u001b[31m\u001b[1m (5/7) Missing option \"--o-representative-sequences\".  (\"--output-dir\" may\n",
      "  also be used)\u001b[0m\n",
      "\u001b[31m\u001b[1m (6/7) Missing option \"--o-denoising-stats\".  (\"--output-dir\" may also be\n",
      "  used)\u001b[0m\n",
      "\u001b[31m\u001b[1m (7/7) Got unexpected extra argument (\\)\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime dada2 denoise-paired \\"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Usage: \u001b[34mqiime dada2 denoise-paired\u001b[0m [OPTIONS]\n",
      "\n",
      "  This method denoises paired-end sequences, dereplicates them, and filters\n",
      "  chimeras.\n",
      "\n",
      "\u001b[1mInputs\u001b[0m:\n",
      "  \u001b[34m\u001b[4m--i-demultiplexed-seqs\u001b[0m ARTIFACT \u001b[32mSampleData[PairedEndSequencesWithQuality]\u001b[0m\n",
      "                         The paired-end demultiplexed sequences to be\n",
      "                         denoised.                                  \u001b[35m[required]\u001b[0m\n",
      "\u001b[1mParameters\u001b[0m:\n",
      "  \u001b[34m\u001b[4m--p-trunc-len-f\u001b[0m INTEGER\n",
      "                         Position at which forward read sequences should be\n",
      "                         truncated due to decrease in quality. This truncates\n",
      "                         the 3' end of the of the input sequences, which will\n",
      "                         be the bases that were sequenced in the last cycles.\n",
      "                         Reads that are shorter than this value will be\n",
      "                         discarded. After this parameter is applied there must\n",
      "                         still be at least a 20 nucleotide overlap between the\n",
      "                         forward and reverse reads. If 0 is provided, no\n",
      "                         truncation or length filtering will be performed\n",
      "                                                                    \u001b[35m[required]\u001b[0m\n",
      "  \u001b[34m\u001b[4m--p-trunc-len-r\u001b[0m INTEGER\n",
      "                         Position at which reverse read sequences should be\n",
      "                         truncated due to decrease in quality. This truncates\n",
      "                         the 3' end of the of the input sequences, which will\n",
      "                         be the bases that were sequenced in the last cycles.\n",
      "                         Reads that are shorter than this value will be\n",
      "                         discarded. After this parameter is applied there must\n",
      "                         still be at least a 20 nucleotide overlap between the\n",
      "                         forward and reverse reads. If 0 is provided, no\n",
      "                         truncation or length filtering will be performed\n",
      "                                                                    \u001b[35m[required]\u001b[0m\n",
      "  \u001b[34m--p-trim-left-f\u001b[0m INTEGER\n",
      "                         Position at which forward read sequences should be\n",
      "                         trimmed due to low quality. This trims the 5' end of\n",
      "                         the input sequences, which will be the bases that\n",
      "                         were sequenced in the first cycles.      \u001b[35m[default: 0]\u001b[0m\n",
      "  \u001b[34m--p-trim-left-r\u001b[0m INTEGER\n",
      "                         Position at which reverse read sequences should be\n",
      "                         trimmed due to low quality. This trims the 5' end of\n",
      "                         the input sequences, which will be the bases that\n",
      "                         were sequenced in the first cycles.      \u001b[35m[default: 0]\u001b[0m\n",
      "  \u001b[34m--p-max-ee-f\u001b[0m NUMBER    Forward reads with number of expected errors higher\n",
      "                         than this value will be discarded.     \u001b[35m[default: 2.0]\u001b[0m\n",
      "  \u001b[34m--p-max-ee-r\u001b[0m NUMBER    Reverse reads with number of expected errors higher\n",
      "                         than this value will be discarded.     \u001b[35m[default: 2.0]\u001b[0m\n",
      "  \u001b[34m--p-trunc-q\u001b[0m INTEGER    Reads are truncated at the first instance of a\n",
      "                         quality score less than or equal to this value. If\n",
      "                         the resulting read is then shorter than `\u001b[4mtrunc-len-f\u001b[0m`\n",
      "                         or `\u001b[4mtrunc-len-r\u001b[0m` (depending on the direction of the\n",
      "                         read) it is discarded.                   \u001b[35m[default: 2]\u001b[0m\n",
      "  \u001b[34m--p-chimera-method\u001b[0m TEXT \u001b[32mChoices('consensus', 'pooled', 'none')\u001b[0m\n",
      "                         The method used to remove chimeras. \"none\": No\n",
      "                         chimera removal is performed. \"pooled\": All reads are\n",
      "                         pooled prior to chimera detection. \"consensus\":\n",
      "                         Chimeras are detected in samples individually, and\n",
      "                         sequences found chimeric in a sufficient fraction of\n",
      "                         samples are removed.           \u001b[35m[default: 'consensus']\u001b[0m\n",
      "  \u001b[34m--p-min-fold-parent-over-abundance\u001b[0m NUMBER\n",
      "                         The minimum abundance of potential parents of a\n",
      "                         sequence being tested as chimeric, expressed as a\n",
      "                         fold-change versus the abundance of the sequence\n",
      "                         being tested. Values should be greater than or equal\n",
      "                         to 1 (i.e. parents should be more abundant than the\n",
      "                         sequence being tested). This parameter has no effect\n",
      "                         if \u001b[4mchimera-method\u001b[0m is \"none\".           \u001b[35m[default: 1.0]\u001b[0m\n",
      "  \u001b[34m--p-n-threads\u001b[0m INTEGER  The number of threads to use for multithreaded\n",
      "                         processing. If 0 is provided, all available cores\n",
      "                         will be used.                            \u001b[35m[default: 1]\u001b[0m\n",
      "  \u001b[34m--p-n-reads-learn\u001b[0m INTEGER\n",
      "                         The number of reads to use when training the error\n",
      "                         model. Smaller numbers will result in a shorter run\n",
      "                         time but a less reliable error model.\n",
      "                                                            \u001b[35m[default: 1000000]\u001b[0m\n",
      "  \u001b[34m--p-hashed-feature-ids\u001b[0m / \u001b[34m--p-no-hashed-feature-ids\u001b[0m\n",
      "                         If true, the feature ids in the resulting table will\n",
      "                         be presented as hashes of the sequences defining each\n",
      "                         feature. The hash will always be the same for the\n",
      "                         same sequence so this allows feature tables to be\n",
      "                         merged across runs of this method. You should only\n",
      "                         merge tables if the exact same parameters are used\n",
      "                         for each run.                         \u001b[35m[default: True]\u001b[0m\n",
      "\u001b[1mOutputs\u001b[0m:\n",
      "  \u001b[34m\u001b[4m--o-table\u001b[0m ARTIFACT \u001b[32mFeatureTable[Frequency]\u001b[0m\n",
      "                         The resulting feature table.               \u001b[35m[required]\u001b[0m\n",
      "  \u001b[34m\u001b[4m--o-representative-sequences\u001b[0m ARTIFACT \u001b[32mFeatureData[Sequence]\u001b[0m\n",
      "                         The resulting feature sequences. Each feature in the\n",
      "                         feature table will be represented by exactly one\n",
      "                         sequence, and these sequences will be the joined\n",
      "                         paired-end sequences.                      \u001b[35m[required]\u001b[0m\n",
      "  \u001b[34m\u001b[4m--o-denoising-stats\u001b[0m ARTIFACT \u001b[32mSampleData[DADA2Stats]\u001b[0m\n",
      "                                                                    \u001b[35m[required]\u001b[0m\n",
      "\u001b[1mMiscellaneous\u001b[0m:\n",
      "  \u001b[34m--output-dir\u001b[0m PATH      Output unspecified results to a directory\n",
      "  \u001b[34m--verbose\u001b[0m / \u001b[34m--quiet\u001b[0m    Display verbose output to stdout and/or stderr\n",
      "                         during execution of this action. Or silence output if\n",
      "                         execution is successful (silence is golden).\n",
      "  \u001b[34m--citations\u001b[0m            Show citations and exit.\n",
      "  \u001b[34m--help\u001b[0m                 Show this message and exit.\n",
      "\n",
      "\u001b[33m                  There were some problems with the command:                  \u001b[0m\n",
      "\u001b[31m\u001b[1m (1/4) Invalid value for \"--output-dir\": 'work/DADA2_denoising_output3'\n",
      "  already exists, will not overwrite.\u001b[0m\n",
      "\u001b[31m\u001b[1m (2/4) Missing option \"--o-table\".  (\"--output-dir\" may also be used)\u001b[0m\n",
      "\u001b[31m\u001b[1m (3/4) Missing option \"--o-representative-sequences\".  (\"--output-dir\" may\n",
      "  also be used)\u001b[0m\n",
      "\u001b[31m\u001b[1m (4/4) Missing option \"--o-denoising-stats\".  (\"--output-dir\" may also be\n",
      "  used)\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime dada2 denoise-paired \\\n",
    "--i-demultiplexed-seqs paired-end-trimmed-demux.qza \\\n",
    "--p-trunc-len-f 160 \\\n",
    "--p-trunc-len-r 150 \\\n",
    "--p-n-threads 0 \\\n",
    "--output-dir work/DADA2_denoising_output3 \\\n",
    "--verbose"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: work/DADA2_denoising_output3/denoising_stats.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime metadata tabulate \\\n",
    "--m-input-file work/DADA2_denoising_output3/denoising_stats.qza \\\n",
    "--o-visualization work/DADA2_denoising_output3/denoising_stats.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-_seyjzzx')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: a038cd41-11cb-4ec1-b431-284d232c3d6d>"
      ]
     },
     "execution_count": 95,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load(\"work/DADA2_denoising_output3/denoising_stats.qzv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: work/DADA2_denoising_output3/rep_seqs.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime feature-table tabulate-seqs \\\n",
    "--i-data work/DADA2_denoising_output3/representative_sequences.qza \\\n",
    "--o-visualization work/DADA2_denoising_output3/rep_seqs.qzv "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-gf2fb14d')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 181aa0f2-0489-4b33-8a5e-e13ca9715f10>"
      ]
     },
     "execution_count": 101,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load(\"work/DADA2_denoising_output3/rep_seqs.qzv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 102,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: work/DADA2_denoising_output3/table.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime feature-table summarize \\\n",
    "--i-table work/DADA2_denoising_output3/table.qza \\\n",
    "--o-visualization work/DADA2_denoising_output3/table.qzv "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-4yzjnkh1')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 80a4bf57-5cef-48a5-a47f-31e7b8efcdd6>"
      ]
     },
     "execution_count": 103,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load(\"work/DADA2_denoising_output3/table.qzv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--2020-06-24 17:54:31--  https://data.qiime2.org/2019.10/common/silva-132-99-515-806-nb-classifier.qza\n",
      "Resolving data.qiime2.org... 52.35.38.247\n",
      "Connecting to data.qiime2.org|52.35.38.247|:443... connected.\n",
      "HTTP request sent, awaiting response... 302 FOUND\n",
      "Location: https://s3-us-west-2.amazonaws.com/qiime2-data/2019.10/common/silva-132-99-515-806-nb-classifier.qza [following]\n",
      "--2020-06-24 17:54:32--  https://s3-us-west-2.amazonaws.com/qiime2-data/2019.10/common/silva-132-99-515-806-nb-classifier.qza\n",
      "Resolving s3-us-west-2.amazonaws.com... 52.218.213.0\n",
      "Connecting to s3-us-west-2.amazonaws.com|52.218.213.0|:443... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 188926431 (180M) [binary/octet-stream]\n",
      "Saving to: ‘silva-132-99-515-806-nb-classifier.qza’\n",
      "\n",
      "silva-132-99-515-80 100%[===================>] 180.17M  6.45MB/s    in 41s     \n",
      "\n",
      "2020-06-24 17:55:13 (4.41 MB/s) - ‘silva-132-99-515-806-nb-classifier.qza’ saved [188926431/188926431]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "! wget https://data.qiime2.org/2019.10/common/silva-132-99-515-806-nb-classifier.qza\n",
    "! mv silva-132-99-515-806-nb-classifier.qza work/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Taxonomy] to: work/classified_sequences2/classification.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime feature-classifier classify-sklearn \\\n",
    "--i-classifier work/silva-132-99-515-806-nb-classifier.qza \\\n",
    "--i-reads work/DADA2_denoising_output3/representative_sequences.qza \\\n",
    "--output-dir work/classified_sequences2 \\\n",
    "--verbose"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: work/classified_sequences2/taxonomy.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime metadata tabulate \\\n",
    "--m-input-file work/classified_sequences2/classification.qza \\\n",
    "--o-visualization work/classified_sequences2/taxonomy.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 108,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-gst00qys')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 86c71f7e-cf91-424c-b27f-b59dbfb17dd5>"
      ]
     },
     "execution_count": 108,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load(\"work/classified_sequences2/taxonomy.qzv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir work/phylogeny2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[AlignedSequence] to: work/phylogeny2/aligned-rep-seqs2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime alignment mafft \\\n",
    "--i-sequences work/DADA2_denoising_output3/representative_sequences.qza \\\n",
    "--o-alignment work/phylogeny2/aligned-rep-seqs2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 115,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[AlignedSequence] to: work/phylogeny2/masked-aligned-rep-seqs2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime alignment mask \\\n",
    "--i-alignment work/phylogeny2/aligned-rep-seqs2.qza \\\n",
    "--o-masked-alignment work/phylogeny2/masked-aligned-rep-seqs2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 116,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Phylogeny[Unrooted] to: work/phylogeny2/fasttree-tree2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime phylogeny fasttree \\\n",
    "--i-alignment work/phylogeny2/masked-aligned-rep-seqs2.qza \\\n",
    "--o-tree work/phylogeny2/fasttree-tree2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 117,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Phylogeny[Rooted] to: work/phylogeny2/fasttree-tree-rooted2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime phylogeny midpoint-root \\\n",
    "--i-tree work/phylogeny2/fasttree-tree2.qza \\\n",
    "--o-rooted-tree work/phylogeny2/fasttree-tree-rooted2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir export2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 121,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported work/DADA2_denoising_output3/table.qza as BIOMV210DirFmt to directory work/export2/table\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime tools export \\\n",
    "--input-path work/DADA2_denoising_output3/table.qza \\\n",
    "--output-path work/export2/table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "metadata": {},
   "outputs": [],
   "source": [
    "! biom convert \\\n",
    "-i work/export2/table/feature-table.biom \\\n",
    "-o work/export2/table/table.tsv --to-tsv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported work/DADA2_denoising_output3/representative_sequences.qza as DNASequencesDirectoryFormat to directory work/export2/rep-seqs.fasta\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "! qiime tools export \\\n",
    "  --input-path work/DADA2_denoising_output3/representative_sequences.qza \\\n",
    "  --output-path work/export2/rep-seqs.fasta"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported work/classified_sequences/taxonomy.qzv as Visualization to directory export2/taxonomy/TSVTaxonomyDirectoryFormat\u001b[0m\n"
     ]
    }
   ],
   "source": [
    " ! qiime tools export \\\n",
    "  --input-path work/classified_sequences/taxonomy.qzv \\\n",
    "  --output-path export2/taxonomy/TSVTaxonomyDirectoryFormat"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported work/phylogeny2/fasttree-tree-rooted2.qza as NewickDirectoryFormat to directory work/export2/exported-tree\u001b[0m\n"
     ]
    }
   ],
   "source": [
    " ! qiime tools export \\\n",
    "  --input-path work/phylogeny2/fasttree-tree-rooted2.qza \\\n",
    "  --output-path work/export2/exported-tree"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running recursive pre-scan... pre-scan complete... transferring data...\n"
     ]
    }
   ],
   "source": [
    "! iput -r work/export2/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
